reverse_proxy/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: install nginx
+  become: true
+  apt:
+    name:
+      - nginx-extras # for proxying
+      - git # for testing
+
+- name: remove default site configuration
+  become: true
+  file:
+    path: /etc/nginx/sites-enabled/default
+    state: absent
+
+- include_tasks: cran.yml
+- include_tasks: cran_test.yml
```

Comment:
Could the content of `cran_test.yml` go in the molecule `converge.yml` playbook? It seems a bit strange to have a test playbook as part of the role.

Reply:
Not sure how I ended up thinking that was a good idea 😓